pkg/kubelet/dockershim/helpers_linux_test.go

```diff
@@ -55,26 +58,26 @@ func TestGetSeccompSecurityOpts(t *testing.T) {
 }
 
 func TestLoadSeccompLocalhostProfiles(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "seccomp-local-profile-test")
+	assert.NoError(t, err)
```

Comment:
shouldn't this be a fatal error?
FYI style guide recommends against assert helpers for exactly this reason, they make it easy to write imprecise tests

Reply:
I think the style guide recommends against useless error messages. In this case, you can still write
```go
require.NoError(t, err, "actual useful error message")
```
I personally don't think that's worse than
```go
if err != nil {
   t.Fatalf("actual useful error message: %s",  err)
}
```

Reply:
I don't have a very strong opinion on this. 
Some of the things assert libs do is very nice.
Probably the use of assert libs should be discussed in the community.
Until then, fine IMO if people do what they prefer.
Definitely not going to block this PR on it.

Reply:
@mtaufen thanks. I think both the `t.Fatal` and `assert.NoError` are accepted now in kubernetes. They are both existing in many packages.

Reply:
s/assert/require

assert is not fatal and the test will continue.